### PR TITLE
Style

### PR DESCRIPTION
**Pull request by almathew**
I'm going to throw this PR up before the Terraform PR just to get feedback, since this is my first foray into this project.

This is largely stylistic changes, although I am making a choice to have Endpoints exclusively operate on Services.
This is going to make Terraform a bit more annoying, but it does mean that we know explicitly what Service is intended.
This annoyance gets us two things:  Services, and Endpoints for an explicit service. 

Sorry, I had originally separated out the style changes from the logic changes, and then :shrug: the commit was lost to the wind, and so the second commit is still a combination of both. I don't think it's that unmanageable, but if it is, let me know and I can figure something out.

### CODE REVIEW

File: aptible/database.go [hunk 1]
```diff
@@ -26,167 +28,139 @@ type DBCreateAttrs struct {
 	DiskSize      int64
 }
 
-func (c *Client) CreateDatabase(env_id int64, attrs DBCreateAttrs) (Database, error) {
+func (c *Client) CreateDatabase(accountId int64, attrs DBCreateAttrs) (Database, error) {
```
Comment (robertfairhead) [inline aptible/database.go:31]: The formal go style for this is accountID based on the [official style guide](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). I tend to follow it for consistency sake but I'm not super bothered about it if you like `Id` better.

Comment (almathew) [reply aptible/database.go:31]: :+1: that's fine. I'll update to that. I thought I was following in Terraform's steps here, but on reflection I was not, so no idea what I was thinking

**Comment by almathew** [comment]
Ok, `ID` is fixed. 

One question: what's the expectation around running tests? Should this actually be connecting to API? Is anything required for that to work properly?

**Comment by robertfairhead** [comment]
The generated code is tested in deploy api when the swagger is generated, so I decided we didn't need to retest that here. I asked Rachel to focus the tests here on helpers, abstractions, etc. that are local so we can try to avoid having to stand up a copy of API for tests all the time. 

The terraform adapter acceptance tests do exercise it against a live instance, so once we run that daily that'll also point to any regressions/deviations here.

**Pull request closed without merging**